.github/workflows/ci.yml

```
@@ -0,0 +1,67 @@
+name: CI
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)